test/TestImport.hs

```diff
@@ -0,0 +1,105 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE QuasiQuotes #-}
+module TestImport
+    ( withApp
+    , runDB
+    , authenticateAs
+
+    -- Convenience exports
```

Comment:
No, it's this:

```haskell
module Foo
    ( foo
    , bar
    , module X
    )

import Bar as X
import Baz as X
import Bat as X
```

Vs:

```hs
module Foo
    ( foo
    , bar
    , module Bar
    , module Baz
    , module Bat
    )

import Bar
import Baz
import Bat
```

Benefit would be less churn when changing what you re-export, downside would be you need to look at the `import`s to see what that `X` means. Keeping them grouped can help, so it's might be fine.